components/mgmt/azure-publc-dns.tf

```diff
@@ -0,0 +1,5 @@
+module "public-dns" {
```

Comment:
typo in the file name 
`components/mgmt/azure-publc-dns.tf`

(publc)

Reply:
Corrected